src/winetricks

```diff
@@ -867,6 +897,9 @@ w_read_key()
             pl*) _W_keymsg="Proszę podać klucz dla programu '${W_PACKAGE}'"
                 _W_nokeymsg="Nie podano klucza"
                 ;;
+            pt*)  _W_keymsg="Por favor, insira a chave do aplicativo '${W_PACKAGE}'"
```

Comment:
Please follow the existing whitespace convention (one space after the paren, not two).

(Similar issues below).